unity-renderer/Assets/Rendering/Culling/Interfaces/CullingControllerSettings.cs

```diff
@@ -18,7 +18,7 @@ public class CullingControllerSettings
 
         public float enableAnimationCullingDistance = 7.5f;
 
-        public int ignoredLayersMask = LayerMask.GetMask("Tutorial", "CharacterPreview");
+        public int ignoredLayersMask = LayerMask.GetMask("Tutorial", "CharacterPreview", "AvatarLOD");
```

Comment:
The new layer is needed to ignore the LODs by the Culling Controller. Do you suggest I use `Tutorial` or `CharacterPreview` existent layers for the LODs? sounds a bit counterintuitive, also I need the main camera to be able to render the LODs layer so the `CharacterPreview` wouldn't be an option. I just checked all the other existent layers and none looks intuitive to be used for the LODs, and if I reuse other existent layer I'd have to add it to this ignored layers list and that would mess up the culling on other objects using those existent layers.

I made sure the new layer is not colliding with anything.

Reply:
Oh, that makes sense, thanks!

Reply:
If this layer is only used for the culling controller ( and we don't plan to filter anything by avatar lods layer), I think we should call it "CullingIgnore" because there is a maximum of 32 layers and it is more likely that there is something more that we want to ignore in the culling controller

Reply:
true that, done!